### PR TITLE
sparkling-campari-orange

### PR DESCRIPTION
**Pull request by vlx01**
Adding FESTIVE AND REFRESHING SPARKLING CAMPARI ORANGE COCKTAIL

### CODE REVIEW

File: src/recipes/sparkling-campari-orange.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            "quantity": "2",
+            "measure": "ounce",
+            "ingredient": "orange juice(no pulp)"
+        },
+        {
+            "quantity": "1",
+            "measure": "ounce",
+            "ingredient": "sparkling water"
+        },
```
Comment (alfg) [inline src/recipes/sparkling-campari-orange.json:25]:
```suggestion
        }
```

File: src/recipes/sparkling-campari-orange.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    ],
+    "directions": [
+        "Fill a glass with ice.",
+        "Pour campari, pour orange juice, pour sparkling water on top.",
+        "Mix and enjoy."
+    ],
+    "image": "Sparkling-Campari-Orange.jpeg",
```
Comment (alfg) [inline src/recipes/sparkling-campari-orange.json:32]:
```suggestion
    "image": "sparkling-campari-orange.jpg",
```

**Review by alfg — CHANGES_REQUESTED** [review]
H @vlx01i, thanks for the contribution! I suggested two fixes. Thanks!

**Review by vlx01 — COMMENTED** [review]
Thankyou @alfg for the suggestions. I have reviewed and made the required changes.

File: src/recipes/sparkling-campari-orange.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            "measure": "ounce",
+            "ingredient": "sparkling water"
+        }
+    ],
+    "directions": [
+        "Fill a glass with ice.",
+        "Pour campari, pour orange juice, pour sparkling water on top.",
+        "Mix and enjoy."
+    ],
+    "image": "Sparkling-Campari-Orange.jpg",
```
Comment (alfg) [inline src/recipes/sparkling-campari-orange.json:32]:
```suggestion
    "image": "sparkling-campari-orange.jpg",
```

Case sensitive. Thanks!

**Review by alfg — CHANGES_REQUESTED** [review]

**Comment by alfg** [comment]
Hi @vlx01, one more fix for case sensitivity. Thanks!

**Comment by vlx01** [comment]
> Hi @vlx01, one more fix for case sensitivity. Thanks!

Reviewed and updated. Thanks, @alfg !! :)

**Comment by alfg** [comment]
Almost there. The other two cases need to be lowercase too.

```
    "image": "sparkling-Campari-Orange.jpg",
    "image": "sparkling-campari-crange.jpg",
```

**Comment by vlx01** [comment]
> Almost there. The other two cases need to be lowercase too.
> 
> ```
>     "image": "sparkling-Campari-Orange.jpg",
>     "image": "sparkling-campari-orange.jpg",
> ```

Oops, my bad. Reviewed and changed. Thanks again.

**Comment by alfg** [comment]
Perfect! Thanks!

**Pull request merged**